srcpkgs/sysstat/template

```diff
@@ -1,12 +1,12 @@
 # Template file for 'sysstat'
 pkgname=sysstat
 version=12.7.6
-revision=1
+revision=2
 build_style=gnu-configure
 configure_args="--enable-copy-only --disable-file-attr
- --with-systemdsystemunitdir= --enable-install-cron"
+ --with-systemdsystemunitdir= --enable-install-cron --enable-sensors"
 conf_files="/etc/default/sysstat /etc/default/sysstat.ioconf"
-hostmakedepends="pkg-config gettext"
+hostmakedepends="pkg-config gettext libsensors-devel"
```

Comment:
Wouldn't this go in `makedepends`, not `hostmakdepends`?

Reply:
`hostmakedepends` is for things that need to run on the host, `makedepends` is  libraries on the target host; this matters most for cross-builds. So your fix would work (though not intentionally) for native builds, but not cross-builds like we do for ARM.

Reply:
I tested cross compiling and running on aarch64.  It works as expected with the current config.  Thanks for catching the `depends` issue @abenson.